src/dune

```diff
@@ -5,5 +5,5 @@
  (public_name reason-react)
  (preprocess
   (pps melange.ppx reason-react-ppx))
- (wrapped false)
+ (wrapped true)
```

Comment:
I have been changes to be `true` @davesnx . I think is incomplete yet? could you give me suggestions for this changes?

Reply:
One way to move forward with this would be to fix the build